authors/apps/articles/filter.py

```diff
@@ -0,0 +1,25 @@
+from django_filters import rest_framework as filters
+from rest_framework.generics import ListAPIView
+from .serializers import ArticleSerializer
+from .models import Article
+
+
+class ArticleFilter(filters.FilterSet):
+    author = filters.CharFilter(field_name='author__username',
```

Comment:
Would it be better to use the name?

Reply:
I think so, the other option would be the authors ID.